app-arch/pet2tgz/pet2tgz-0.3.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2012 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=4
+EAPI=7
```

Comment:
If updating EAPI, you must also revbump.

Reply:
done